multiqc/utils/mqc_colour.py

```diff
@@ -382,6 +384,18 @@ def get_colours(self, name="GnBu"):
                 "#fddaec",
                 "#f2f2f2",
             ],
+            "Highcharts": [
```

Comment:
Lowercase?
```suggestion
            "highcharts": [
```

Reply:
Or maybe nicer to drop the name "Highcharts" completely - they have updated their default colour palette since this, plus I guess we will continue to use these colours in Plotly..

```suggestion
            # Originally from Highcharts
            "plot_defaults": [
```